tremor-script/src/interpreter.rs

```diff
@@ -674,7 +674,55 @@ where
     Ok(())
 }
 
+/// enum mirroring `PatchOperation` carrying evaluated elements
+/// as we need to evaluate expressions in patch operations
+/// in one go, before we do the actual in-place manipulations
+/// in order to not expose temporary states of the patched object to intermittent operations
+/// example:
```

Comment:
This example isn't very clear. It does not even appear to be valid Tremor Script syntax according to https://docs.tremor.rs/tremor-script/#patch.

Reply:
good catch, i was just quickly sketching that down.